chasm/lib/activity/proto/v1/activity_state.proto

```diff
@@ -123,7 +123,7 @@ message ActivityAttemptState {
     google.protobuf.Timestamp last_started_time = 3;
 
     // The time when the last activity attempt completed. If activity has not been completed yet, it will be null.
-    google.protobuf.Timestamp last_attempt_complete_time = 4;
+    google.protobuf.Timestamp last_complete_time = 4;
```

Comment:
Just `started_time` and `completed_time` IMHO because this is information of a single attempt. And we would want to be consistent with the tense.